src/test/java/org/redcrosswarriors/emailservice/TestRequest.java

```diff
@@ -0,0 +1,22 @@
+package org.redcrosswarriors.emailservice;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class TestRequest {
+
+    @Test
+    void request_constructor_test(){
```

Comment:
Need to change this to use cammelCase for the method name since were programming in Java. 
I'm getting the sense that your used to programming in python or C based on the naming conventions you've been using.